environment/coverage.yml

```diff
@@ -4,29 +4,23 @@ channels:
   - numba
   - intel
   - numba/label/dev
-  - pkgs/main
+  - conda-forge
   - nodefaults
 dependencies:
-  - python=3.9
-  - defaults:libffi
+  - libffi
   - gxx_linux-64
   - dpcpp_linux-64
-  - numba 0.57*
-  - dppy/label/dev:dpctl
-  - dppy/label/dev:dpnp
+  - numba=0.57
+  - dpctl
+  - dpnp
   - spirv-tools
   - dpcpp-llvm-spirv
   - packaging
   - pytest
-  - pip
-  - pip:
-    - coverage
-    - pre-commit
-    - flake8
-    - black==20.8b1
-    - pytest-cov
-    - pytest-xdist
-    - pexpect
-variables:
-  CHANNELS: -c defaults -c numba -c intel -c numba/label/dev -c dppy/label/dev --override-channels
-  CHANNELS_DEV: -c dppy/label/dev -c defaults -c numba -c intel -c numba/label/dev --override-channels
+  - coverage
+  - pre-commit
```

Comment:
Why do we really need the linter packages here?

Reply:
It was just a copy from pip to conda package. Removed. Looks like it is redundant here.

Reply:
:+1: